### ISSUE TITLE
OneDrive for Business token error

### ISSUE THREAD

**Issue by LudekK** [KoenZomers/KeePassOneDriveSync#78]
Hallo Koen,
I am facing the same pron¨blem as last year (ticket 51]. There is token error when I want to used database synchro on OneDrive for Business. See attachemt please.
![error](https://user-images.githubusercontent.com/36100331/51374440-4f40a800-1b03-11e9-87f7-79cd6b0a2395.JPG)

**Comment by Heloiseboanerges** [comment]
Hi Koen
I'm getting the same error when sync with OneDrive, I've tryed with the last release but I'm still getting the same error...
![image](https://user-images.githubusercontent.com/40353290/51388533-005a3900-1b2a-11e9-956a-1147340db4e9.png)

**Comment by KoenZomers** [comment]
Sorry guys I can't seem to reproduce this problem making it impossible for me to troubleshoot. Is this a new sync not working or an existing sync that stopped working?

**Comment by Heloiseboanerges** [comment]
Hi Koen, In my case it is an existing sync that stopped working.

**Comment by Heloiseboanerges** [comment]
Hi Koen,
We found a solution by deleting the file KeePass.config from c:/users/myuser/AppData and from C:\Program Files(x86)\KeePassPasswordSafe2. We sync the database again, but instead of usin the OneDrive for Business we used the Graph, and now the DB sync is working. 
Thank you!

**Comment by yeags47** [comment]
> Hi Koen,
> We found a solution by deleting the file KeePass.config from c:/users/myuser/AppData and from C:\Program Files(x86)\KeePassPasswordSafe2. We sync the database again, but instead of usin the OneDrive for Business we used the Graph, and now the DB sync is working.
> Thank you!

This solution worked for me as well - thanks so much!

**Comment by KoenZomers** [comment]
Thanks for sharing your workaround @Heloiseboanerges! Simply deleting the entry through KeePass -> Options -> KeeOneDriveSync Options should actually also have achieved the same result.

Also not sure why I could not reproduce this before as I can now. It exists in the underlying OneDrive API as well and it caused by an API update by Microsoft leading to a different behavior of handling the OAuth2 tokens. I have updated the OneDrive API package and tested this with KeePass OneDriveSync and this issue is now resolved. I'm waiting for the updated NuGet Package to show up in the feed so I can release version 2.0.2.3 of this plugin in which this issue will be resolved.
